### ISSUE TITLE
onNavigationStateChange Equivalent

### ISSUE THREAD

**Issue by juanpaternina** [iou90/react-native-autoheight-webview#32]
Hello first of all nice work!!.

I'm implementing this on my project but onNavigationStateChange doesn't exist in the project, there is a onNavigationStateChange equivalent?

Tks!!

**Comment by caseyg1204** [comment]
@juanpaternina did you find a work around for this?

**Comment by iou90** [comment]
onNavigationStateChange no exist? what version of react-native-autoheight-webview you used and what version of your react-native?

**Comment by Barylskyigb** [comment]
There's no way to pass onNavigationStateChange to the react-native-autoheight-webviews WebView. I would like to submit a PR to support this - it's easy for ios, but in case of Android I see that the library is using a custom WebView component called RCTAutoHeightWebView. I don't know much about how react-native works with Java/Android code and have no idea how could I pass onNavigationStateChange function to RCTAutoHeightWebView. @iou90, maybe you could help/make a PR for this?

**Comment by Barylskyigb** [comment]
I need this function to handle link in html provided to AutoHeightWebview and later open it in an external browser, something like this: https://stackoverflow.com/a/40382325/4896751

**Comment by Barylskyigb** [comment]
I was able to implement link handling with below code:

```
render() {
    const { html } = this.props;

    return (
      <WebView
        ref={webview => (this.webview = webview)}
        source={{ html }}
        onLoadStart={({ nativeEvent }) => {
          const { url } = nativeEvent;

          if (url.startsWith('http')) {
            if (Linking.canOpenURL(url)) {
              this.webview.stopLoading();
              Linking.openURL(url);
            }
          }
        }}
      />
    );
  }
```

(also had to add stopLoading functions for ios/android in AutoHeightWebView, #37)

So maybe @juanpaternina and @caseyg1204 could use onLoadStart as an alternative to onNavigationStateChange

**Comment by iou90** [comment]
@Barylskyigb merged stopLoading at 0.5.0
